web/views/analyses.py

```diff
@@ -72,7 +72,7 @@ def index(self):
         page = int(request.args.get('page', 1))
 
         analyses = current_user.analyses.find().sort('_id', DESCENDING).limit(PER_PAGE).skip((page - 1) * PER_PAGE)
-        pagination = Pagination(page=page, per_page=PER_PAGE, total=analyses.count(), css_framework='bootstrap3')
+        pagination = Pagination(page=page, per_page=PER_PAGE, total=current_user.analyses.estimated_document_count(), css_framework='bootstrap3')
```

Comment:
Unfortunately this is still not the same. This would return the count for the whole collection instead of with the applied filters.